packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -52,8 +50,7 @@ const AppConfig = ({children, locals = {}}) => {
     }
 
     const commerceApiConfig = locals.appConfig.commerceAPI
-
-    const appOrigin = getAppOrigin()
+    const appOrigin = useAppOrigin()
```

Comment:
Let's make sure that the corresponding files in pwa-kit-create-app are also updated.